docs/en-us/sidecarset.md

```diff
@@ -278,6 +278,72 @@ spec:
 ```
 **Note: If you use Scatter, it is recommended to set only a pair of key-values for scatter. It will be easier to understand.**
 
+### Hot Upgrade Sidecar
+Sidecar's in-place upgrade method is to stop the old version of the container first and then create the new version of the container, this upgrade method is more suitable for stateless sidecar containers, e.g. LogTail.
+
+But for many stateful sidecar containers, this upgrade method cannot meet the demand, e.g. Istio Envoy. Envoy, as a proxy container in Pod, proxies all the traffic, and if restarted directly will affect the availability of business services, so we try to find a way to meet this stateful sidecar container upgrade solution.
```

Comment:
But for many stateful sidecar containers, e.g. Istio Envoy, this upgrade method is problematic. Envoy, as a proxy container in the Pod, proxies all the traffic, and if restarted directly, the availability of service is affected. so we provide a new solution for such stateful sidecar container upgrade.